cls/SourceControl/Git/WebUIDriver.cls

```diff
@@ -218,27 +218,61 @@ ClassMethod HandleRequest(pagePath As %String, InternalName As %String = "", Out
                 } else {
                     set inFile = ""
                 }
+                // Want to invoke merge conflict autoresolver in case of issues
+                if (($piece(args, " ", 1, 2)  = "stash pop") || ($piece(args, " ", 1, 2)  = "stash apply")) {
```

Comment:
Over time we should be moving from /git to /git-command. Wondering about whether it makes sense to make that move for stash now and put this handling there.

Reply:
Also wondering about whether we should take the extra time to solve the general problem and cover merge (via WebUI) as well.

Reply:
At minimum we should refactor "run git command and auto-resolve conflicts" to its own method used from everywhere it's relevant.